crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -238,6 +245,9 @@ where
             metrics: Metrics::default(),
         };
 
+        // Optimistically assume that the network state is idle from the start.
+        this.sync_state_updater.update_sync_state(SyncState::Idle);
```

Comment:
while this is technically true, this could cause some issues if launched without CL because then the pipeline isn't triggered.

this should be moved to a new function that 
calls `sync.set_pipeline_sync_target` and also sets the sync state to syncing.

Reply:
what are the issues that can be caused by this call? 

> this should be moved to a new function that
calls sync.set_pipeline_sync_target and also sets the sync state to syncing.

this is already handled in `on_sync_event` fn

Reply:
the issue is that if we set it to idle by default and don't have a CL that sends us FCU we will never trigger the pipeline, but the state is still marked at idle.

eth_syncing will be incorrect and txpool wastes resources inserting into the pool